### PR TITLE
Create an action to check if closed issue has release notes

### PR DESCRIPTION
**Pull request by ov7a**

### CODE REVIEW

File: dist/release-notes/index.js
Comment (ljacomet) [inline dist/release-notes/index.js:1]: ❓ Is this file supposed to be committed?
If yes, can you confirm it is generated and not needing a review?

File: src/release-notes.ts [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+    !labels.includes(hasNotesLabel) //in case notes are updated in an unrelated PR
+}
+
+// see https://github.com/orgs/community/discussions/24492
+// and https://github.com/orgs/community/discussions/24367#discussioncomment-3243930
+async function getLinkedPrNumbers(context: Context, issue: any): Promise<number[]> {
+  const response = await fetch(`https://github.com/${context.repo.owner}/${context.repo.repo}/issues/${issue.number}`)
+  const html = await response.text()
+  const document = cheerio.load(html)
+  const prNumbers = Array.from(document('development-menu form span a'))
```
Comment (ljacomet) [inline src/release-notes.ts:33]: 😨 Yikes, that sounds ... brittle ... but I guess there is no better way for now 😢

**Review by ljacomet — COMMENTED** [review]
Very high level review, the logic makes sense.

But man, what it is built on is sketchy 😞

File: src/release-notes.ts (hunk 1 above)
Comment (ov7a) [reply src/release-notes.ts:33]: Exactly :harold:

**Comment by ov7a** [comment]
Testing:
1. Note-worty issue is closed with a PR having changes - https://github.com/ov7a/issue-management-test/issues/29 - should be left as closed with label added and label removed
2. Note-worty issue is closed without a PR - https://github.com/ov7a/issue-management-test/issues/30 - should be reopened with label added, mentioning asignee
3. Note-worty issue with a has: label - https://github.com/ov7a/issue-management-test/issues/32 - should be skipped
4. Not worthy issue -  https://github.com/ov7a/issue-management-test/issues/15 - should be skipped

**Pull request merged**